suite-common/redux-utils/src/createReducerWithExtraDeps.ts

```diff
@@ -7,19 +7,20 @@ type NotFunction<T> = T extends Function ? never : T;
 
 type ExtraDependenciesForReducer = Pick<ExtraDependencies, 'actionTypes' | 'actions' | 'reducers'>;
 
-export const createReducerWithExtraDependencies =
-    <S extends NotFunction<any>>(
-        initialState: S | (() => S),
-        builderCallback: (
-            builder: ActionReducerMapBuilder<S>,
-            extra: ExtraDependenciesForReducer,
-        ) => void,
-    ) =>
-    (extraDeps: ExtraDependencies) =>
+export const createReducerWithExtraDeps = <S extends NotFunction<any>>(
+    initialState: S | (() => S),
+    builderCallback: (
+        builder: ActionReducerMapBuilder<S>,
+        extra: ExtraDependenciesForReducer,
+    ) => void,
+) => {
+    const prepareReducer = (extraDeps: ExtraDependencies) =>
         createReducer(initialState, builder =>
             builderCallback(builder, {
                 actionTypes: extraDeps.actionTypes,
                 actions: extraDeps.actions,
                 reducers: extraDeps.reducers,
             }),
         );
+    return { prepareReducer };
```

Comment:
please revert this changes, this was mistake it should return prepare function directly without any wrapping into object